x/profiles/keeper/msg_server_app_link.go

```diff
@@ -97,12 +97,6 @@ func (k MsgServer) UnlinkApplication(
 			sdk.NewAttribute(sdk.AttributeKeyAction, sdk.MsgTypeURL(msg)),
 			sdk.NewAttribute(sdk.AttributeKeySender, msg.Signer),
 		),
-		sdk.NewEvent(
-			types.EventTypeApplicationLinkDeleted,
-			sdk.NewAttribute(types.AttributeKeyUser, msg.Signer),
-			sdk.NewAttribute(types.AttributeKeyApplicationName, msg.Application),
-			sdk.NewAttribute(types.AttributeKeyApplicationUsername, msg.Username),
-		),
```

Comment:
It is duplicated with the event inside `DeleteApplicationLink`:
https://github.com/desmos-labs/desmos/blob/405f77bbe774ff3d2dd264f0f929959ffd1151cf/x/profiles/keeper/keeper_app_links.go#L109